class.jetpack.php

```diff
@@ -4544,24 +4544,14 @@ public static function register() {
 		);
 		$response = Jetpack_Client::_wp_remote_request( Jetpack::fix_url_for_bad_hosts( Jetpack::api_url( 'register' ) ), $args, true );
 
-
 		// Make sure the response is valid and does not contain any Jetpack errors
-		$valid_response = Jetpack::init()->validate_remote_register_response( $response );
-		if( is_wp_error( $valid_response ) || !$valid_response ) {
-		    return $valid_response;
+		$json = Jetpack::init()->validate_remote_register_response( $response );
+		if( is_wp_error( $json ) || ! $json ) {
+		    return $json;
 		}
 
-		// Grab the response values to work with
-		$code   = wp_remote_retrieve_response_code( $response );
-		$entity = wp_remote_retrieve_body( $response );
-
-		if ( $entity )
-			$json = json_decode( $entity );
-		else
-			$json = false;
-
 		if ( empty( $json->jetpack_secret ) || ! is_string( $json->jetpack_secret ) )
```

Comment:
Since we're in the code, can we go ahead and add brackets, `{ }1, around this conditional?